apps/src/blockly/addons/blocklyThemes/cdoCVDThemes.js

```diff
@@ -0,0 +1,58 @@
+import GoogleBlockly from 'blockly/core';
+import DeuteranopiaTheme from '@blockly/theme-deuteranopia';
+import TritanopiaTheme from '@blockly/theme-tritanopia';
+
+// styles for color vision deficiency themes
+const CVDStyles = {
+  // colors chosen from 15-color or 24-color palette for colorblindness at http://mkweb.bcgsc.ca/colorblind/palettes.mhtml#15-color-palette-for-colorbliness
+  default: {
+    colourPrimary: '00DCB5' // aquamarine
+  },
+  behavior_blocks: {
+    colourPrimary: '#00E307' // radioactive green
+  },
+  dance_blocks: {
+    colourPrimary: '#9400E6' // veronica
+  },
+  event_blocks: {
+    colourPrimary: '#008169' // deep sea
+  },
+  music_blocks: {
+    colourPrimary: '#FFCFE2' // azalea
+  },
+  sprite_blocks: {
+    colourPrimary: '#F60239' // tractor red
+  },
+  setup_blocks: {
+    colourPrimary: 'FFAC3B' // frenzee
+  },
+  world_blocks: {
+    colourPrimary: '#450270' // christalle
+  }
+};
+
+const cdoDeuteranopiaStyles = {
+  ...DeuteranopiaTheme.blockStyles,
+  ...CVDStyles
+};
+
+const cdoTritanopiaStyles = {
```

Comment:
in your screenshot Deuteranopia and Tritanopia look the same, is that correct or should there be a different screenshot in the description?

Reply:
The two themes are very similar! We selected colors for [customized CDO blocks](https://github.com/code-dot-org/code-dot-org/blob/afd384937abd637de506050e4da8e0b9138a30a0/apps/src/blockly/addons/blocklyThemes/cdoCVDThemes.js#L6-L32) that were similar to current style colors from [this color palette](http://mkweb.bcgsc.ca/colorblind/palettes.mhtml#24-color-palette-for-colorbliness). For other block categories (such as loop blocks), the theme colors are a bit different as you can see with the `repeat every ... ` block in the screenshots. 
We plan to consult with @moneppo to see if we want to differentiate these themes more or just use one color vision deficiency theme instead of two. FYI -  deuteranopia is inability to perceive 'green' light and tritanopia is inability to perceive 'blue' light - [ref].(https://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/#:~:text=People%20with%20protanopia%20are%20unable,blues%20and%20yellows%20stand%20out.)